core/src/main/java/org/geysermc/geyser/bedrock/camera/CameraUtil.java

```diff
@@ -0,0 +1,80 @@
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Geyser
+ */
+
+package org.geysermc.geyser.bedrock.camera;
+
+import org.cloudburstmc.protocol.bedrock.data.camera.CameraAudioListener;
+import org.cloudburstmc.protocol.bedrock.data.camera.CameraPreset;
+import org.cloudburstmc.protocol.common.DefinitionRegistry;
+import org.cloudburstmc.protocol.common.NamedDefinition;
+import org.cloudburstmc.protocol.common.SimpleDefinitionRegistry;
+import org.cloudburstmc.protocol.common.util.OptionalBoolean;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class CameraUtil {
+
+    public static final DefinitionRegistry<NamedDefinition> CAMERA_DEFINITIONS;
+
+    public static final List<CameraPreset> CAMERA_PRESETS;
+
+    static {
+        CAMERA_PRESETS = List.of(
+                new CameraPreset("minecraft:first_person", "", null, null, null, null, OptionalBoolean.empty()),
+                new CameraPreset("minecraft:free", "", null, null, null, null, OptionalBoolean.empty()),
+                new CameraPreset("minecraft:third_person", "", null, null, null, null, OptionalBoolean.empty()),
+                new CameraPreset("minecraft:third_person_front", "", null, null, null, null, OptionalBoolean.empty()),
+                new CameraPreset("geyser:free_audio", "minecraft:free", null, null, null, CameraAudioListener.PLAYER, OptionalBoolean.of(false)),
+                new CameraPreset("geyser:free_effects", "minecraft:free", null, null, null, CameraAudioListener.CAMERA, OptionalBoolean.of(true)),
+                new CameraPreset("geyser:free_audio_effects", "minecraft:free", null, null, null, CameraAudioListener.PLAYER, OptionalBoolean.of(true)));
+
+        CAMERA_DEFINITIONS = SimpleDefinitionRegistry.<NamedDefinition>builder()
+                .addAll(IntStream.range(0, CAMERA_PRESETS.size())
+                        .mapToObj(i -> CameraDefinition.of(CAMERA_PRESETS.get(i).getIdentifier(), i))
+                        .collect(Collectors.toList()))
+                .build();
+    }
+
+    public static NamedDefinition getById(int id) {
+        return CAMERA_DEFINITIONS.getDefinition(id);
+    }
+
+    public static NamedDefinition getByFunctionality(boolean audio, boolean effects) {
```

Comment:
if these are the only methods and they are named this then i would consider naming this class `CameraDefinitions`